src/app/shared/select/select-constraint-item/select-constraint-item.component.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Lumeer.io, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Component, ChangeDetectionStrategy, Input, Output, EventEmitter} from '@angular/core';
+import {AttributesResource} from '../../../core/model/resource';
+import {ConstraintConfig} from '../../../core/model/data/constraint';
+import {I18n} from '@ngx-translate/i18n-polyfill';
+
+export interface SelectConstraintItemId {
+  resourceIndex: number;
+  attributeId: string;
+}
+
+@Component({
+  selector: 'select-constraint-item',
+  templateUrl: './select-constraint-item.component.html',
+  styleUrls: ['./select-constraint-item.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SelectConstraintItemComponent {
```

Comment:
This is a very confusing name for this set of components, pipes and interfaces. If I understand it correctly, they have nothing to do with `SelectConstraint`.

Reply:
Ups, somehow I forgot about `SelectConstraint`. I will change it, but I'm not very good at naming classes, interfaces etc :D